pom.xml

```diff
@@ -153,6 +159,37 @@
         </executions>
       </plugin>
 
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>unpack</id>
+            <goals>
+              <goal>unpack</goal>
+            </goals>
+            <phase>process-classes</phase>
```

Comment:
change to `process-resources`
This ensures that the file is unpacked before compilation and then it also works when running `mvn hpi:run` for interactive testing.